src/routes/faqs.tsx

```diff
@@ -1,5 +1,267 @@
+2. Find the “Staked MASK” card from the previous period, and click on the "Reinvest MASK" button.
+3. Confirm the reinvestment into the new staking period.
+4. After on-chain confirmation, these MASK will be reinvested into the new staking period and earn points.
+5. Reinvestment requires only one step, "Reinvest", without the need to "Unstake" and then "Stake", saving gas fees.
+
+
+### What are the benefits of staking MASK?
+1. **Earning rewards**: Staking MASK allows you to earn token rewards from high-quality investment projects.
+2. **Long-term holding incentive**: Staking MASK encourages long-term token holding. With possible staking durations, you will be incentivized to lock your MASK tokens for a period of time, reducing frequent trading and potentially increasing long-term investment returns.
+
+
+### What are the risks of staking MASK?
+1. **Market risk**: The market value of MASK tokens may fluctuate, which can affect the value of your staked tokens. If the token price decreases, the value of your staked tokens may decrease.
+2. **Contract risk**: Interacting with the staking contract for MASK tokens involves inherent risks associated with smart contracts. There is a possibility of contract vulnerabilities, misconfigurations, or attacks that could result in the loss of your staked funds.
+3. **Unstake restrictions**: During the lock-up period of staked funds, you cannot unstake or withdraw your tokens. This may limit the flexibility of using your funds.
+4. **Uncertain returns**: Staking MASK may lead to returns, but the specific rate of return is uncertain. The returns can be influenced by changes in the staking platform, market factors, and the behavior of network participants.
+5. Mask Network Team is driven to mitigate the above risks and eliminate them entirely to the extent possible. Despite this, they may still exist.
+
+## **Smart Contract**
+
+### MASK Contract 
+1. Chain name: ETH network
+2. Address: 0x69af81e73a73b40adf4f3d4223cd9b1ece623074
+
+
+### TON Contract 
+1. Chain name: ETH network
+2. Address: 0x582d872a1b094fc48f5de31d3b73f2d9be47def1
+
+
+### RSS3 Contract
+1. Chain name: ETH network
+2. Address: 0xc98d64da73a6616c42117b582e832812e7b8d57f
+
+
+### Stake Contract 
+1. Test: 0xece3ef2bf6f6fa7f13beab519c60a72e92bbd47c
+
+
+### Reward Contract 
+1. Test: 0xf0c196d1b1489738cda956e994e82ef6897e85bc
+
+
+## **Other**
+
+### How to buy MASK？
+
+You can buy MASK spot on major CEX such as Binance, OKX, and Gate.io .`
+  const contentElRef = useRef<HTMLDivElement>(null)
+
+  return (
+    <Center
+      w="100%"
+      pt={{ base: '110px', md: '80px' }}
+      px={{ base: '24px', xl: '48px' }}
+      transition="200ms"
+      overflow="auto"
+    >
+      <Flex
+        w="100%"
+        maxW="maxW"
+        pt={{ base: 0, md: '64px' }}
+        position="relative"
+        direction={{ base: 'column', md: 'row' }}
+      >
+        <FaqsToc contentRef={contentElRef} />
+        <Box
+          ref={contentElRef}
```

Comment:
Passing a callback to ref instead, you can avoid using useEffect inside the TOC component. 

```suggestion
          ref={el => handle（el)}
```